entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Where is this file used?

Reply:
https://github.com/jurajpiar/rskj/actions/runs/9059773191/workflow#L149-L170